about.html

```diff
@@ -114,33 +115,30 @@ <h2>2021</h2>
           </div>
         </div>
      </section>
-     <section id="about-footer">
+     <section id="about-partners">
         <div class="advert">
           <h6>Partners</h6>
           <hr>
           <div class="advlogo">
             <h2>Mozilla</h2>
             <h3>Google</h3>
             <h2>NAVER</h2>
-            <p>daumkakao</p><img src="./images/Icons/clip.png" alt="clip">airbnb</p>
+            <p>daumkakao<img src="./images/Icons/clip.png" alt="clip">airbnb</p>
           </div>
         </div>
-        </div>
       </section>
      <section id="about-footer">
```

Comment:
- [ ]  Kindly as mentioned before try to respect the original design and make your footer similar to the original design by providing some padding and reducing text font-size 

Current design            |  original design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/163587528-5c15a6be-4113-4c69-a59c-4425b42874d3.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/163587452-096e30fe-1fff-4ebb-804e-85d02e15353f.PNG)